src/SPC/builder/LibraryBase.php

```diff
@@ -294,6 +294,16 @@ public function beforePack(): void
         // do something before pack, default do nothing. overwrite this method to do something (e.g. modify pkg-config file)
     }
 
+    public function patchBeforeConfigure(): bool
+    {
+        return false;
+    }
+
+    public function patchBeforeMake(): bool
```

Comment:
I think it would be better if we add a PHPDoc to provide instructions to avoid confusion with the Extension.